proto/BUILD.bazel

```diff
@@ -126,3 +137,15 @@ py_grpc_library(
     srcs = [":p4runtime_proto"],
     deps = [":p4runtime_py_proto"],
 )
+
+build_test(
```

Comment:
should CI build this target?

cc @smolkaj

Reply:
I think what you said in the meeting is correct. Looks like it would already be built with the existing workflows: https://github.com/p4lang/p4runtime/blob/af4847d6ca4f785a84e4002d4cb158fc0284ce05/.github/workflows/ci-build-proto.yml#L42-L43

Reply:
Yes +1

Reply:
Note that CI will build, but not run (i.e. test) the target.
If we wanted to do that, we should change line 43 to:
```
       run: cd proto && bazel build //... && bazel test //...
```

Reply:
Done!